tests/target/configs/format_cargo_toml/true/dotted-key/Cargo.toml

```
@@ -32,3 +32,7 @@ ac = 1
 
 [a.z]
 az = 1
+
+# Very long (from std’s Cargo.toml)
+[target.'cfg(any(all(target_family = "wasm", target_os = "unknown"), target_os = "xous", all(target_vendor = "fortanix", target_env = "sgx")))' .dependencies]
```

You were probably aware of this, but there is an extra white space here:
```suggestion
[target.'cfg(any(all(target_family = "wasm", target_os = "unknown"), target_os = "xous", all(target_vendor = "fortanix", target_env = "sgx")))'.dependencies]
```